<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_12">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Рабочая папка:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="workingDir">
       <property name="statusTip">
        <string>Путь к рабочей папке, в которой будут развернуты библиотека gamebase и использующие ее приложения</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="chooseWorkingDir">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/icons/Folder.png</normaloff>:/images/icons/Folder.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Папка для загрузок:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="downloadsDir">
       <property name="statusTip">
        <string>Путь к рабочей папке, в которой будут развернуты библиотека gamebase и использующие ее приложения</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="chooseDownloadsDir">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/icons/Folder.png</normaloff>:/images/icons/Folder.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Путь к vcvarsall.bat:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="vcVarsPath">
       <property name="statusTip">
        <string>Путь к рабочей папке, в которой будут развернуты библиотека gamebase и использующие ее приложения</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="chooseVCVarsPath">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/icons/Folder.png</normaloff>:/images/icons/Folder.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Список источников библиотеки gamebase</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_15">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTableView" name="gamebaseSources">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="addGamebaseServerButton">
            <property name="statusTip">
             <string>Добавить сервер, на котором будут появляться новые версии библиотеки gamebase</string>
            </property>
            <property name="text">
             <string>Добавить сервер</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addGamebaseDirectoryButton">
            <property name="statusTip">
             <string>Добавить папку, в которой будут появляться новые версии библиотеки gamebase</string>
            </property>
            <property name="text">
             <string>Добавить папку</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeGamebasePathButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="statusTip">
             <string>Удалить выбранный источник библиотеки gamebase</string>
            </property>
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Список источников приложений</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_14">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QTableView" name="appSources">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QPushButton" name="addAppDirectoryButton">
            <property name="statusTip">
             <string>Добавить папку, в которой будут появляться новые приложения, использующие gamebase</string>
            </property>
            <property name="text">
             <string>Добавить папку</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeAppPathButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="statusTip">
             <string>Удалить выделенный источник приложений</string>
            </property>
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="acceptButton">
       <property name="statusTip">
        <string>Применить изменения в настройках</string>
       </property>
       <property name="text">
        <string>Применить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="statusTip">
        <string>Сбросить изменения в настройках</string>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
